journeyapps-db/src/types/primitives/Datetime.ts

```diff
@@ -0,0 +1,42 @@
+import { DatetimeType as SchemaDatetimeType } from '@journeyapps/parser-schema';
+import { DBTypeMixin } from '../Type';
+import moment = require('moment');
+
+export class DatetimeType extends DBTypeMixin(SchemaDatetimeType) {
+  valueToJSON(value: any) {
+    if (value instanceof Date) {
+      return value.toISOString();
+    } else {
+      return null;
+    }
+  }
+
+  valueFromJSON(value: string) {
+    if (typeof value == 'string') {
+      const parsed = Date.parse(value);
+      if (isNaN(parsed)) {
+        return null;
+      } else {
+        return new Date(parsed);
+      }
+    } else {
+      return null;
+    }
+  }
+
+  format(value: any, format: string = 'MMMM D YYYY h:mm A'): string {
```

Comment:
would be better to put this in a static variable in this class called FORMAT_DEFAULT